MiniApp/Classes/Protocols/MiniAppMessageDelegate.swift

```diff
@@ -1,8 +1,11 @@
+@available(*, deprecated, message: "protocol renamed to MiniAppMessageDelegate")
+public typealias MiniAppMessageProtocol = MiniAppMessageDelegate
+
 /**
 Public Protocol that will be used by the Mini App to communicate
  with the Native implementation
 */
-public protocol MiniAppMessageProtocol: class {
+public protocol MiniAppMessageDelegate: class {
```

Comment:
Won't this be a breaking change for the Host App because they have to rename it? Is it possible to keep `MiniAppMessageProtocol` for now and deprecate it?

I guess you would also need to add a deprecated `MiniApp.create` method for accepting `MiniAppMessageProtocol`.

Reply:
No this won't be breaking change because I created a `typeAlias` :

```swift
@available(*, deprecated, message: "protocol renamed to MiniAppMessageDelegate")
public typealias MiniAppMessageProtocol = MiniAppMessageDelegate
```

I tested it without changing anything in the sample host app, everything is retro compatible

Reply:
Ah, okay nice! Sorry, I didn't see that.